OrbitLinuxTracing/MakeUniqueForOverwrite.h

```diff
@@ -0,0 +1,20 @@
+#ifndef ORBIT_LINUX_TRACING_MAKE_UNIQUE_FOR_OVERWRITE_H_
+#define ORBIT_LINUX_TRACING_MAKE_UNIQUE_FOR_OVERWRITE_H_
+
+#include <memory>
+
+// Unlike std::make_unique, calls new without parentheses, causing the object
+// to be default-initialized instead of value-initialized.
+// This can prevent useless memory initialization.
+// Standard version coming in C++20.
```

Comment:
check _­_­cplusplus value here so that this is disabled for C++20?

Reply:
Well, this is not in the std namespace, so if it's used in the code it is still needed even with C++20. Unless I forward the call to std::make_unique_for_overwrite for C++20, but this shouldn't be done until we have a final standard.

Reply:
AFAIK no standard library has P1020 implemented yet. I would argue that writing this "blindly" will introduce more problems than it tries to fix, when we switch to C++20 eventually.